tests/unit/connector/test_http_input.py

```diff
@@ -315,17 +314,6 @@ def test_get_next_with_hmac_of_raw_message(self):
         connector_next_msg, _ = connector.get_next(1)
         assert connector_next_msg == expected_event, "Output event with hmac is not as expected"
 
-    @pytest.mark.parametrize("endpoint", ["/auth-json-secret", "/.*/[A-Z]{2}/json$"])
-    def test_endpoint_has_credentials(self, endpoint, credentials_file_path):
-        mock_env = {ENV_NAME_LOGPREP_CREDENTIALS_FILE: credentials_file_path}
-        with mock.patch.dict("os.environ", mock_env):
-            new_connector = Factory.create({"test connector": self.CONFIG}, logger=self.logger)
-            new_connector.pipeline_index = 1
-            new_connector.setup()
-            endpoint_config = new_connector.http_server.endpoints_config.get(endpoint)
-            assert endpoint_config.credentials.username, endpoint
-            assert endpoint_config.credentials.password, endpoint
-
```

Comment:
because the http_server implementation does not have the endpoints_config anymore. So `new_connector.http_server.endpoints_config.get(endpoint)` won't work.

Additionally the behaviour should be tested and not the implementation. The next test checks if basic auth works. This test is not necessary.

Reply:
Ah I missed that the server doesn't have the endpoints_config anymore. Thanks.